frontend/src/pages/premium.module.scss

```diff
@@ -82,6 +82,16 @@
       margin: 0 auto;
     }
 
+    .pill {
+      background-color: $color-violet-30;
```

Comment:
(The force push just now was to change this from -20 to -30 as per Eduardo's request :smile: )